addons/mail_plugin/tests/test_res_partner_iap.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import psycopg2
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tools import mute_logger
+
+
+class TestResPartnerIap(MailCommon):
+    def test_compute_iap_enrich_info(self):
+        partner = self.partner_employee
+
+        self.assertFalse(partner.iap_enrich_info)
+
+        partner_iap = self.env["res.partner.iap"].create({"partner_id": partner.id, "iap_enrich_info": "test info"})
+        partner.invalidate_cache()
+        self.assertEqual(partner.iap_enrich_info, "test info")
+
+        partner_iap.unlink()
+        partner.iap_enrich_info = "test info 2"
+
+        partner_iap = self.env["res.partner.iap"].search([("partner_id", "=", partner.id)])
+        self.assertTrue(partner_iap, "Should have create the <res.partner.iap>")
+        self.assertEqual(partner_iap.iap_enrich_info, "test info 2")
+
+        partner.iap_enrich_info = "test info 3"
+        partner_iap.invalidate_cache()
+        new_partner_iap = self.env["res.partner.iap"].search([("partner_id", "=", partner.id)])
+        self.assertEqual(new_partner_iap, partner_iap, "Should have write on the existing one")
+        self.assertEqual(partner_iap.iap_enrich_info, "test info 3")
+
+    def test_tes_partner_iap_creation(self):
```

Comment:
test_res_partner_iap_create ? (not test_tes :D )

Reply:
oups :/